### PR TITLE
Add Cancel Button Props Object for iOS

### PR DESCRIPTION
**Pull request by M-Jas**
#What does this PR do?

- Gives the user the ability to change/update the text color of the Search Bar's cancel button.

### CODE REVIEW

**Comment by M-Jas** [comment]
@xavier-villelegier to confirm are you talking about something like the "loadingProps" for the searchBar?

**Comment by xavier-villelegier** [comment]
@M-Jas Yea exactly. It's just to avoid adding too many `cancelButtonXXX` props

**Comment by M-Jas** [comment]
@xavier-villelegier you got it. New PR should be up there shortly. Thanks for the feedback.

File: src/badge/badge.js [hunk 1]
```diff
@@ -35,9 +35,9 @@ const Badge = props => {
   let Component = View;
 
   if (component) {
-    Component = component
+    Component = component;
   } else if (onPress) {
-    Component = TouchableOpacity
+    Component = TouchableOpacity;
```
Comment (xavier-villelegier) [inline src/badge/badge.js]: Just curious, is it your own prettier or the pre-commit hook which added these ?

Comment (M-Jas) [reply src/badge/badge.js]: Must be the pre-commit, I didn't step into those file.

**Comment by M-Jas** [comment]
@xavier-villelegier Are we good on this or is there anything else I need to do?

**Pull request merged**